homeassistant/components/ihc/manual_setup.py

```diff
@@ -81,11 +80,7 @@ def validate_name(config):
 )
 
 SENSOR_SCHEMA = DEVICE_SCHEMA.extend(
-    {
-        vol.Optional(
-            CONF_UNIT_OF_MEASUREMENT, default=UnitOfTemperature.CELSIUS
-        ): cv.string
-    }
+    {vol.Optional(CONF_UNIT_OF_MEASUREMENT): cv.string}
```

Comment:
I am concerned that this falls under ADR-0010.
I see that some preliminary work was done for config-flow, but I don't see the follow-up to #64852

Reply:
It's ok in regards to ADR10 as it's not a new option. We're just removing a default.